tests/models/nn/transformers/test_base.py

```diff
@@ -112,6 +113,7 @@ def test_save_load_for_unfitted_model(
         config = {
             "deterministic": True,
             "item_net_block_types": (IdEmbeddingsItemNet, CatFeaturesItemNet),
+            "similarity_module_type": SimilarityModuleDistance,
```

Comment:
This should not be in the tests. Right now you need it only because the default value is not set. Let's set default instead

Reply:
Fixed

Reply:
It is still in many places which makes me think tests would break without it :)
Could you please remove in cases where we just had
```
config = {
  "deterministic": True,
  "item_net_block_types": (IdEmbeddingsItemNet, CatFeaturesItemNet)
}
```

Actually, even "item_net_block_types" are not needed there. We forgot to remove them

Reply:
Sorry, I forgot about other `"similarity_module_type"` in `test_base.py`
Also removed `"item_net_block_types"`